src/main/resources/data.properties

```diff
@@ -8,6 +8,7 @@ products.json=data/products.json
 product.group.seed.json=data/product-group-seed.json
 service.agreements.json=data/service-agreements.json
 legal.entities.with.users.json=data/legal-entities-with-users.json
+arrangements.data.json=data/retail-arrangements.json
```

Comment:
the name is logical but the 'random' arrangements are seeded from `product.group.seed.json`. It would be better to have some name similarity so it is easy to detect their connection

Reply:
fixed.